### ISSUE TITLE
Cosmetic: use PyTorch's build-in scheduler for LR adjustment during training

### ISSUE THREAD

**Issue by mkroutikov** [jiesutd/NCRFpp#90]
Instead of manually tweaking learning rate for SGD, use PyTorch built-in `lr_scheduler`:

https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.StepLR

StepLR or LambdaLR seem appropriate

**Comment by jiesutd** [comment]
Thank you very much for your suggestion! I believe using the `lr_scheduler` would be more elegant than my implementation. Will update it later. 

Thanks again for your useful suggestions! I appreciate.
